README.md

```diff
@@ -56,20 +53,23 @@ let res = dynamo_client
 Creating new span:
 
 ```rust
-// create new span directly by using current span context
-let aws_span = AwsSpan::new(AwsTarget::Dynamo("table_name"), "GetItem");
+// create new span in the current span's context using either a dedicated constructor
+let aws_span = DynamodbSpanBuilder::get_item("table_name").start();
+// or a generic one
+let aws_span = AwsSpanBuilder::dynamodb("GetItem", vec!["table_name"]).start();
```

Comment:
@Travispersson I renamed `AwsOperation` to `AwsSpanBuilder` to make it clearer what it does